inst/assets/BS4/pkgdown.css

```diff
@@ -465,3 +465,8 @@ summary {
 details p {
   margin-top: -.5rem;
 }
+
+/* tabsets */
```

Comment:
Another tweak that might make sense would be more padding/margin at the top of tab content.

Reply:
Is there normally some styling around the content of the tabs? For the pills example in particular, it's hard to tell how the tab names are related to the content.

Reply:
I added left and bottom borders for the pills example. Its color won't be hard-coded forever, since the blslib variables PR will prevent this kind of hard-coding.